### ISSUE TITLE
Pluto TV Anime All Day

### ISSUE THREAD

**Issue by youngbluespring** [iptv-org/iptv#5444]
**Channel Name:** Pluto TV Anime All Day (US)
**Country:** US
**Language:** English
**Website:** 
**Stream URL (optional):** xxx
**Notes (optional):** I got a US VPN and the Anime All Day channel airs Love Hina but the link on iptv-org airs some card-themed anime. Wondering which feed this came from

**Comment by LaneSh4d0w** [comment]
There's multiple versions of "Anime All Day" depending on the region.

**Comment by AstaFatherSatan** [comment]
Check out this directory. Has a lot of free epg and m3u like for Plutotv, pbs, etc. Region sectioned as well... https://i.mjh.nz/PlutoTV/all.m3u8 (all Pluto tv regions) https://i.mjh.nz/PlutoTV/all.xml (xml for it)

**Comment by what-a** [comment]
I reviewed "Pluto TV Anime All Day" in us_pluto.m3u and found some issues;
First problem is one of the streams was actually "Pluto TV Anime All Ages." I just submitted a pull request to clean that up.
Next problem is that the UK and France streams are lumped in with the USA
- URLs containing "channel/5812b7d3249444e05d09cc49" are the USA Anime All Day stream (no issue here).
- URLs containing "channel/5c363ac9e41be30cb6054c7c" are the UK Anime All Day stream and need to be moved.
- URLs containing "channel/5f8ed52a62fa750007733239" are the France Anime All Day stream and need to be moved.
Where should the UK and France streams move to? Looks like we don't have uk_pluto.m3u nor fr_pluto.m3u list.
